packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-batcher.ts

```diff
@@ -78,12 +81,26 @@ const CallDiagnosticEventsBatcher = Batcher.extend({
       case 'client.media-engine.ready':
         joinTimes.totalMediaJMT = cdl.getTotalMediaJMT();
         break;
+
+      case 'client.mediaquality.event':
+        audioSetupDelay.joinRespRxStart = cdl.getAudioJoinRespRxStart();
```

Comment:
just to double check: you have a jira or another PR for fixing these function, right?